identity-service/src/routes/healthCheck.js

```diff
@@ -177,7 +177,11 @@ module.exports = function (app) {
     let balances = []
 
     // run fundRelayerIfEmpty so it'll auto top off any accounts below the threshold
-    await fundRelayerIfEmpty()
+    try {
+      await fundRelayerIfEmpty()
+    } catch (err) {
+      return errorResponseServerError({ error: err.toString() })
```

Comment:
lets make this a error log so it runs the rest of the function? and returns the balances?